src/bashi/results.py

```diff
@@ -827,3 +833,56 @@ def _remove_all_rocm_images_older_than_ubuntu2004_based(
             value_name2=HIPCC,
             value_version2=ANY_VERSION,
         )
+
+
+def _remove_unsupported_cuda_versions_for_ubuntu(
+    parameter_value_pairs: List[ParameterValuePair],
+    removed_parameter_value_pairs: List[ParameterValuePair],
+):
+    remove_parameter_value_pairs_ranges(
+        parameter_value_pairs,
+        removed_parameter_value_pairs,
+        parameter1=UBUNTU,
+        value_name1=UBUNTU,
+        value_min_version1=20.04,
+        value_min_version1_inclusive=True,
+        value_max_version1=9999,
+        parameter2=ALPAKA_ACC_GPU_CUDA_ENABLE,
+        value_name2=ALPAKA_ACC_GPU_CUDA_ENABLE,
+        value_min_version2=OFF,
+        value_min_version2_inclusive=False,
+        value_max_version2=11,
+        value_max_version2_inclusive=True,
```

Comment:
```suggestion
        value_max_version2_inclusive=False,
```
Needs to be exclusive, because otherwise you remove the pair Ubuntu 20.04 + CUDA 11.0.
Please also the pair to the tests. This corner case is not covered by your test casese. Only by the long running test.